doc/api/ooni/orchestrate.md

```diff
@@ -0,0 +1,233 @@
+
+The `production_registry_url`, `testing_registry_url`, `production_events_url`,
+and `testing_events_url` functions return, respectively, the production and
+testing URLs of the orchestrator's registry and events web services used by
+OONI.
+
+Once you have constructed an orchestration client, you can configure it by
+setting the proper configuration attributes, described below:
+
+The `logger` attribute can be override to use a custom logger. By default, the
+global MeasurementKit logger is used.
+
+The `settings` attribute allows to specify optional settings. By default, an
+empty settings object is used, meaning that defaults will always be used.
+
+The `available_bandwidth` attribute indicates how much bandwidth you would
+like to use for automatic tests run through orchestration. By default, this
+attribute is empty, meaning that no bandwidth quota will be enforced.
+
+The `device_token` attribute uniquely identifies the app inside a device, and
+is only meaningful in the mobile context. This token is generated by the mobile
+OS for the current application and is used to identify it.
+
+The `events_url` attribute contains the URL used to query the `events` web
+service of the orchestrator. By default, this attribute is set to the production
+events service URL used by OONI.
+
+The `language` attribute tells the orchestrator the language used by
+this probe. By default this attribute is unspecified.
+
+The `network_type` attribute tells the orchestrator the type of the network
+the probe is connected to (i.e. Wi-Fi or mobile). By default this attribute is
+unspecified.
+
+The `geoip_country_path` and `geoip_asn_path` are used by the orchestrator
+client to guess the country and autonomous system number, if they are not
+provided by the API user (see below). By default these attributes are empty,
+meaning that MK will not attempt to geolocate the user.
+
+The `platform` attribute tells the orchestrator the platform in
+which the probe runs (e.g., `android`, `ios`). If not set, this
+attribute will be filled automatically by MeasurementKit when the
+actual request for the orchestrator is prepared.
+
+The `probe_asn` attribute tells the orchestrator the autonomous
+system number (ASN) of the ISP that the probe is connected to. If
+not set, this attribute will be filled automatically by MeasurementKit
+when the actual request for the orchestrator is prepared.
+
+The `probe_cc` attribute tells the orchestrator the country code
+of the country in which the probe currently is. If not set, this
+attribute will be filled automatically by MeasurementKit when the
+actual request for the orchestrator is prepared.
+
+The `probe_family` attribute allows to bind a set of different
+probes to a common family of probes. This attribute is currently
+reserved for future use.
+
+The `registry_url` attribute contains the URL used to query the
+`registry` web service of the orchestrator. By default, this attribute
```

Comment:
Maybe you can link to: https://github.com/TheTorProject/proteus/tree/master/proteus-registry